### ISSUE TITLE
[FEATURE] EVI: subscribers of an event to expire in a better way

### ISSUE THREAD

**Issue by goharahmed** [OpenSIPS/opensips#2943]
**Is your feature request related to a problem? Please describe.**

Given the expedited resolution of my previous Crash report, I thought maybe expedite the review on a PR using this issue. :) 

1 - The subscribers of any event could only print their static time of subscription i.e 3600 and from output of MI command users can't tell when the subscription will actually expire. or how long this subscription is valid for.
2 - The "expired" subscription to an event continue to print their subscription data with original values when "subscriber_list" MI command is executed.
3 - The expired subscription to an event was ONLY removed when that event is raised. Until the event is raised the MI command keep printing the possibly expired subscription which is incorrect information to the users.
**Describe the solution you'd like**

1 - EVI: prints the subscriber expiry as a countdown value relevant to the subscriber's original expiry time.
2 - EVI: marks the subscriber as "expired" if it goes beyond its expiry time
3 - EVI: remove the expired subscribers from the list if MI command "subscribers_list" is executed.
**Implementation**
 
  - Component:  `evi`  module) 
  - Type:  module function

This following PR contains further details.
Proposes PR: https://github.com/OpenSIPS/opensips/pull/2919

**Describe alternatives you've considered**

**Additional context**

**Comment by razvancrainea** [comment]
I don't think it is necessary to open a different ticket, let's keep all the info in the PR. Unless there's something else missing there and should be treated separately, or not yet implemented in the PR. 
If everything is in there, let's close the ticket and follow up there.
